samples/python/OpenVINO_EP_samples/cleanroom_worker_safety/cleanroom_sample_OV_EP.py

```diff
@@ -0,0 +1,156 @@
+'''
+Copyright (C) 2021, Intel Corporation
+SPDX-License-Identifier: Apache-2.0
+'''
+
+import numpy as np
+import onnxruntime as rt
+import cv2
+import time
+import os
+
+def sigmoid(x, derivative=False):
+  return x*(1-x) if derivative else 1/(1+np.exp(-x))
+
+def softmax(x):
+  scoreMatExp = np.exp(np.asarray(x))
+  return scoreMatExp / scoreMatExp.sum(0)
+
+# color look up table for 5 different classes for cleanroom sample use case
+clut = [(0,0,0),(255,0,0),(255,0,255),(0,0,255),(0,200,0)]
+
+# 5 labels that the tiny-yolov2 model can do the object_detection on
+label = ["unprotected","bunny suit","glasses","head","robot"]
+
+model_file_path = "tiny_yolo_v2_cleanroom.onnx"
```

Comment:
For all the examples, if the model/ input data for the examples aren't included in the sample folders with the proper name, maybe change the value to TODO: Input with your x path so the user knows they need to add their input/model paths.

Reply:
@oliviajain
for the cleanroom python sample, the users have to use the pre-trained .onnx model provided by us. However when I am trying to push it, it is not getting reflected in Msft/onnxruntime repo. so just created a tar file of the .onnx file and pushed it. The users have to use the same video files as provided in the data folder to test this sample. I have added TODO as requested and a clear documentation is already available for the same.

Yes for the other samples, the users have to download the model from ONNX model zoo and have to use their own data to test the sample. I have added TODO as requested. Also we have a clear documentation for all the samples which would tell them that they have to download the models from onnx model zoo and have to use their own data.